litr-muxers/src/main/java/com/linkedin/android/litr/muxers/MediaFormatEx.kt

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright 2023 LinkedIn Corporation
+ * All Rights Reserved.
+ *
+ * Licensed under the BSD 2-Clause License (the "License").  See License in the project root for
+ * license information.
+ *
+ * Author: Ian Bird
+ */
+package com.linkedin.android.litr.muxers
+
+import android.media.MediaFormat
+import com.linkedin.android.litr.MimeType
+import java.nio.ByteBuffer
+
+const val KEY_MIME_TYPE = MediaFormat.KEY_MIME
+const val KEY_BIT_RATE = MediaFormat.KEY_BIT_RATE
+const val KEY_WIDTH = MediaFormat.KEY_WIDTH
+const val KEY_HEIGHT = MediaFormat.KEY_HEIGHT
+const val KEY_CHANNEL_COUNT = MediaFormat.KEY_CHANNEL_COUNT
+const val KEY_SAMPLE_RATE = MediaFormat.KEY_SAMPLE_RATE
+
+fun MediaFormat.isVideo(): Boolean {
```

Comment:
Note: Doesn't have to happen in this PR, but this a very useful class that could be moved into main `litr` module.